### PR TITLE
fix: WrappedValue.unwrap empty string behavior

### PR DESCRIPTION
**Pull request by speigg**
Currently:
```
var value = new WrappedValue('') // passing empty string
console.log(WrappedValue.unwrap(value) === '' ) // prints "false"
```

Now:
```
var value = new WrappedValue('') // passing empty string
console.log(WrappedValue.unwrap(value) === '' ) // prints "true"
```

## PR Checklist

- [ ] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [ ] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [ ] You have signed the [CLA](http://www.nativescript.org/cla).
- [ ] All existing tests are passing: https://github.com/NativeScript/NativeScript/blob/master/DevelopmentWorkflow.md#running-unit-tests.
- [ ] Tests for the changes are included - https://github.com/NativeScript/NativeScript/blob/master/WritingUnitTests.md.

## What is the new behavior?

Fixes/Implements/Closes #[Issue Number].

### CODE REVIEW

File: tns-core-modules/data/observable/observable.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export class WrappedValue implements WrappedValueDefinition {
     }
 
     public static unwrap(value: any) {
-        return (value && value.wrapped) ? value.wrapped : value;
+        return (value && 'wrapped' in value) ? value.wrapped : value;
```
Comment (manoldonev) [inline tns-core-modules/data/observable/observable.ts]: There is a tslint error here -- should be double quotes:
``` bash
> NativeScript@ tslint /home/travis/build/NativeScript/NativeScript
> tslint --config build/tslint.json 'tns-core-modules/**/*.ts' 'tests/**/*.ts' 'apps/**/*.ts' 'e2e/**/*.ts' -e '**/node_modules/**' -e '**/platforms/**'
ERROR: tns-core-modules/data/observable/observable.ts:16:26 - ' should be "
```

**Comment by manoldonev** [comment]
@speigg there are failing unit tests with this change:
```
JS: Test: --- [XML-DECLARATION.test_TabViewHasCorrectParentChain] FAILED: Cannot use 'in' operator to search for 'wrapped' in true, Stack: TypeError: Cannot use 'in' operator to search for 'wrapped' in true
```

You can run the unit tests locally like this:
``` bash
cd tests
tns run android
```

**Comment by vakrilov** [comment]
As @manoldonev  pointed out - the proposed solution fails when `unwrap` is given a boolean value.
Lets just make an `instanceof` check:
```
return (value instanceof WrappedValue) ? value.wrapped : value;
```

**Comment by vakrilov** [comment]
@cla-bot check

**Comment by manoldonev** [comment]
test

**Comment by speigg** [comment]
@manoldonev thanks for taking care of this, I haven't had time to work on this myself

**Comment by manoldonev** [comment]
test

**Pull request merged**